doc/extensibility/IDynamicEnumValuesProvider.md

```diff
@@ -95,6 +96,12 @@ Provides a dynamic list of enum values using an `IDynamicEnumValuesGenerator`. Y
     }
 ```
 
+###Changes coming in Update 1
+`TryCreateEnumValueAsync` will be connected and is called whenever the user submits a new value to the property
+`AllowCustomValues` is now obsolete and will be removed in future versions
+
+When a user submits a value `TryCreateEnumValuesAsync` will be called first with that value. If a task of null is returned CPS will then call `GetListedValuesAsync` and try to match the user supplied value to an existing `IEnumValue`. If that also returns null then the user supplied value will be placed directly into the property field.
```

Comment:
Readability: a comma after "when a user submits a value" would help it read better.
"Task of null" sounds like `Task<null>`. Instead, perhaps say "Task with a null result".
You may want to clarify explicitly that a null Task (no task at all) should *never* be returned.